### PR TITLE
Fix analyzer warnings related to SemanticsListener

### PR DESCRIPTION
**Pull request by abarth**
These were introduced by the recent engine roll.

### CODE REVIEW

**Comment by abarth** [comment]
TBR @Hixie

**Comment by abarth** [comment]
I looked at landing your outstanding "semantics" branch, but it still has some unaddressed review comments.  Hopefully this won't cause you too much merge pain...

File: packages/flutter/lib/src/rendering/semantics.dart [hunk 1]
```diff
@@ -445,4 +445,7 @@ class SemanticsServer extends mojom.SemanticsServer {
   void scrollDown(int nodeID) {
     SemanticsNode.getSemanticActionHandlerForId(nodeID, neededFlag: _SemanticFlags.canBeScrolledVertically)?.handleSemanticScrollDown();
   }
+
```
Comment (Hixie) [inline packages/flutter/lib/src/rendering/semantics.dart]: if you put this at the top of this class it'll make my merge easier

**Comment by Hixie** [comment]
The semantics debugger side of this is gonna suck to merge, but so is life as a programmer. :-)

lgtm

File: packages/flutter/lib/src/rendering/semantics.dart (hunk 1 above)
Comment (abarth) [inline packages/flutter/lib/src/rendering/semantics.dart]: Done.

**Pull request merged**